### PR TITLE
Version/3.11

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,216 +1,84 @@
...
+ * Remove tgz2build directory.
 
 # Notes to the Developers
 
- * Extract template perf test function to testlib
-
- * Print a debug message when logmsg passed to the Python side
-
- * Allow http module (curl) to be build with cmake
+ * Rewrite merge-grammer script from Perl to Python.
```
Comment (presidento) [inline NEWS.md]: typo: merge-gramm`a`r

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

File: NEWS.md (hunk 1 above)
Comment (lbudai) [inline NEWS.md]: thanks! fixed

**Comment by lbudai** [comment]
When travis & kira are green, please merge.

**Comment by kira-syslogng** [comment]
Build SUCCESS, the tests were executed on test branch: master and test suite: functions

**Pull request merged**